lib/embed/viewer/geo.rb

```diff
@@ -10,14 +10,20 @@ def initialize(*args)
       def body_html
         Nokogiri::HTML::Builder.new do |doc|
           doc.div(class: 'sul-embed-body sul-embed-geo', 'style' => "max-height: #{body_height}px", 'data-sul-embed-theme' => "#{asset_url('geo.css')}") do
+            doc.div(id: 'sul-embed-geo-map', 'style' => "height: #{body_height}px") {}
```

Comment:
Line is too long. [86/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.